frontend/src/modules/dashboard/components/dashboard-task.vue

```diff
@@ -202,6 +202,10 @@ const taskCreatePermission = computed(
 )
 
 const storeUnsubscribe = store.subscribeAction((action) => {
```

Comment:
Sorry if I'm lacking context but should the tasks store be handled the same way as integrations? What I mean is that integrations are fetched from the start of the application regardless of opening the Integrations page. Once the tenant switches, the integrations are reset. Shouldn't it be the same for tasks?

Nonetheless, I think I found an issue with this approach which is the number of uncompleted tasks next to the "Tasks" menu item. After switching tenant, that value is only shown once you access the Tasks page. Could you please confirm if it's an issue? 🙏